src/apiActionSteps.ts

```diff
@@ -0,0 +1,120 @@
+import memory from '@qavajs/memory';
+import { When } from '@cucumber/cucumber';
+import { sendHttpRequest } from '@apiHelpers';
+import { RequestInit } from 'node-fetch';
+
+/**
+ * Send request to the API
+ *
+ * @example
+ * I send "GET" request to "{BASE_API_URL}/some_endpoint"
+ *
+ * @param method should be named as one of the http methods (e.g. GET, POST, PUT, DELETE and etc.)
+ * @param url should be API endpoint
+ */
+When('I send {text} request to {landing-url}', async function (method: string, url: string) {
+  const conf: RequestInit = {
+    method,
+  };
+  const response = await sendHttpRequest(url, conf);
+
+  // store response to memory to be able to use it in next steps
+  memory.setValue(`${browser.sessionId}response`, response);
```

Comment:
maybe we need to give user ability to save responce with custom name. And there maybe a case when browser is not available (in pure API) project